schema/defs-image.json

```diff
@@ -11,14 +11,47 @@
       "type": "string",
       "pattern": "^[a-z0-9_+.-]+:[a-f0-9]+$"
     },
+    "elementDescriptor": {
```

Comment:
Do you need this type (which drops `urls`?).  I'd expect you could just use:

    "$ref": "content-descriptor.json"

in the `elements` array (or whatever that ends up being called).